beacon-chain/rpc/eth/beacon/handlers_pool_test.go

```diff
@@ -722,6 +781,11 @@ func TestSubmitAttestations(t *testing.T) {
 			})
 		})
 		t.Run("post-electra", func(t *testing.T) {
+			params.SetupTestConfigCleanup(t)
+			config := params.BeaconConfig()
+			config.ElectraForkEpoch = 0
+			params.OverrideBeaconConfig(config)
+			defer params.SetupTestConfigCleanup(t)
```

Comment:
This doesn't do anything, please stop adding these deferred cleanup calls. You just need to call `SetupTestConfigCleanup` once at the beginning of your test (already happening on line 784). It hooks into the test lifecycle machinery to restore a backup copy of the active and mainnet configs after the test is finished.